Sources_v3/StreamChatUI/ChatChannel/ChatMessageImageGallery.swift

```diff
@@ -7,7 +7,7 @@ import StreamChat
 import UIKit
 
 open class ChatMessageImageGallery<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
-    public var data: Data? {
+    public var data: AttachmentListViewData<ExtraData>? {
         didSet { updateContentIfNeeded() }
     }
```

Comment:
Should we start calling this property `content` on all views?

Reply:
renamed